py_mmd_tools/nc_to_mmd.py

```diff
@@ -892,37 +892,115 @@ def get_metadata_identifier(self, mmd_element, ncin, **kwargs):
                 )
         return naming_authority + ':' + ncid
 
-    def get_related_dataset(self, mmd_element, ncin):
-        """Get id and relation type for a related dataset"""
-        acdd_ext_relation_type = mmd_element['relation_type']['acdd_ext']
-        relation_types = []
-        acdd_ext_relation_type_key = list(acdd_ext_relation_type.keys())[0]
+    def get_related_dataset_OLD(self, mmd_element, ncin):
```

Comment:
No need to keep this. Delete

Reply:
Well - this is the reason for the backward compatibility question. We could call this function if the new one fails...

Reply:
sletter OLD og lager en tydelig commit-melding, så er det lett å hente koden tilbake om nødvendig